### ISSUE TITLE
Remove guns shooting when hitting the ground

### ISSUE THREAD

**Issue by ATPGaming** [studio-minus/ppg-bugs-and-requests#2470]
### Feature description

As the title states, remove guns shooting when hitting the ground. It's just annoying. I'm not suggesting this to be able to be toggled because I don't want the settings menu to be 2 miles long considering you can't search it.

### Feature purpose

I'm not really even sure why it was added in the first place, it just makes the game more annoying to play. I hate it when a drop a gun next to a human and it shoots him. It doesn't really add much to the game besides realism, but it's just extremely irritating. You may say that it's just a minor feature that happens rarely and I shouldn't get worked up over it, but it just gets on my nerves.

**Comment by TeslaCoil9001** [comment]
I agree, I made a factory that spawns and sorts different guns, and when they land in the storage tubes, sometimes they shoot and clog up the system

**Comment by DindinYT37** [comment]
Either it will be toggle-able, or it stays.

**Comment by Drogo-man** [comment]
It's because whomever stores the guns that you spawn, forget to turn the safety on before dispensing them into the chamber.

**Comment by TeslaCoil9001** [comment]
> Either it will be toggle-able, or it stays.

Yes, toggle-able: Able to be toggled.

**Comment by ATPGaming** [comment]
This suggestion is bad, @mestiez please delete this issue

**Comment by DindinYT37** [comment]
> This suggestion is bad, @mestiez please delete this issue

Issues cannot be deleted.
